scripts/west_commands/export.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+from west.commands import WestCommand
+
+from pathlib import PurePath
+from zcmake import run_cmake
+
+
+class ZephyrExport(WestCommand):
+
+    def __init__(self):
+        super().__init__(
+            'zephyr-export',
+            # Keep this in sync with the string in west-commands.yml.
+            'export Zephyr installation as a CMake Config package',
+            'export Zephyr installation as a CMake Config package',
```

Comment:
The third argument should be at least a complete sentence with proper capitalization, if not entire paragraphs. It's what shows up in `west help zephyr-export`. See sign.py for examples.

Reply:
Fixed.

And improved the help text.
```
usage: west zephyr-export [-h]

This command registers the current Zephyr installation as a CMake
Config package in the CMake User Package Registry.

In Windows, the CMake User Package Registry is found in:
HKEY_CURRENT_USER\Software\Kitware\CMake\Packages\

In Linux and MacOS, the CMake User Package Registry is found in:
~/.cmake/packages/
```

Reply:
Thanks! My only feedback is that this Looks Like Unnecessary Capitalization.

Would "CMake config package" and "CMake user package registry" be OK?

Reply:
I used this capitalization and wording, cause that is what CMake does:
https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html#user-package-registry

Reply:
Thanks for the reference.

I see that 3 out of 4 uses of that string in a sentence are lowercase "user package registry" (the section header appears to be in US [title case](https://en.wikipedia.org/wiki/Letter_case#Headings_and_publication_titles)):

![upr](https://user-images.githubusercontent.com/35088/76112466-1ed1fc80-5f97-11ea-83bf-6a534cfb0f54.png)

Reply:
I did notice CMake doc is not consistent in its casing, so I had to choose one way.
So the link was merely for referring why I chose this casing.

If you feel strong for it, Ican change, but when counting 3 out of 4, you should also look at other part, where the also mix cases:
![image](https://user-images.githubusercontent.com/9802583/76199064-779bd200-61ef-11ea-818e-10b362ed0fe0.png)
so I guess we just have to choose which casing we prefer.
I'm open to both ways.

Reply:
Up to you. I'll resolve this comment.

Reply:
Unnecessary Capitalization Removed.